### PR TITLE
add godoc for ResourceSelector

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

### CODE REVIEW

File: pkg/backup/item_action.go [hunk 1]
```diff
@@ -45,9 +45,27 @@ type ResourceIdentifier struct {
 // included/excluded resources, and a label-selector that can be used
 // to match a set of items from a cluster.
 type ResourceSelector struct {
+	// IncludedNamespaces is a slice of namespace names to match. All
+	// namespaces in this slice, except those in ExcludedNamespaces,
+	// will be matched. A nil/empty slice matches all namespaces.
 	IncludedNamespaces []string
+	// ExcludedNamespaces is a slice of namespace names not to match.
```
Comment (ncdc) [inline pkg/backup/item_action.go]: "not to match" -> "to exclude"

File: pkg/backup/item_action.go [hunk 2]
```diff
@@ -45,9 +45,27 @@ type ResourceIdentifier struct {
...
-	IncludedResources  []string
-	ExcludedResources  []string
-	LabelSelector      string
+	// IncludedResources is a slice of resources to match. Resources
+	// may be specified as full names (e.g. "services") or abbreviations
+	// (e.g. "svc"). All resources in this slice, except those in
+	// ExcludedResources, will be matched. A nil/empty slice matches
+	// all resources.
+	IncludedResources []string
+	// ExcludedResources is a slice of resources not to match.
```
Comment (ncdc) [inline pkg/backup/item_action.go]: not to match -> to exclude

File: pkg/restore/item_action.go [hunk 3]
```diff
@@ -35,9 +35,27 @@ type ItemAction interface {
 // included/excluded resources, and a label-selector that can be used
 // to match a set of items from a cluster.
 type ResourceSelector struct {
+	// IncludedNamespaces is a slice of namespace names to match. All
+	// namespaces in this slice, except those in ExcludedNamespaces,
+	// will be matched. A nil/empty slice matches all namespaces.
 	IncludedNamespaces []string
+	// ExcludedNamespaces is a slice of namespace names not to match.
```
Comment (ncdc) [inline pkg/restore/item_action.go]: ditto

File: pkg/restore/item_action.go [hunk 4]
```diff
@@ -35,9 +35,27 @@ type ItemAction interface {
...
-	IncludedResources  []string
-	ExcludedResources  []string
-	LabelSelector      string
+	// IncludedResources is a slice of resources to match. Resources
+	// may be specified as full names (e.g. "services") or abbreviations
+	// (e.g. "svc"). All resources in this slice, except those in
+	// ExcludedResources, will be matched. A nil/empty slice matches
+	// all resources.
+	IncludedResources []string
+	// ExcludedResources is a slice of resources not to match.
```
Comment (ncdc) [inline pkg/restore/item_action.go]: ditto

**Comment by skriss** [comment]
updated

**Pull request merged**